app/contexts/users/queries/user_queries.rb

```diff
@@ -0,0 +1,30 @@
+# Query Object
+module Contexts
+  module Users
+    module Queries
+      class UserQueries
+        attr_reader :users
+
+        def initialize(users = User.all)
+          @users = users
+        end
+
+        def all
+          @users
```

Comment:
please use `attr_reader`
```suggestion
          users
```